dlup/data/dataset.py

```diff
@@ -504,11 +520,12 @@ def parse_rois(rois, image_size, scaling: float) -> tuple[tuple[tuple[int, int],
         if not origin_positive or not image_within_borders:
             raise ValueError(f"ROIs should be within image boundaries. Got {rois}.")
 
-    rois = [
+    _rois = [
         (
-            np.ceil(np.asarray(coords) * scaling).astype(int).tolist(),
+            (np.ceil(np.asarray(coords) * scaling).astype(int)).tolist(),
             np.floor(np.asarray(size) * scaling).astype(int).tolist(),
         )
         for coords, size in rois
     ]
```

Comment:
@jonasteuwen you're not casting anything here. so no need for the initial `_`. Right now the wrong `rois` is returned I assume